### DISCUSSION TITLE
Welcome to nice-node Discussions!

### DISCUSSION THREAD

**Discussion by jgresham** [NiceNode/nice-node#8] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by SpiralBlu** [comment]
Hi...decided to try this NiceNode out on my Linux Box.  Are there any rewards to running this?  I can't see justifying building a box with 2TB+ of storage for it if not.  Currently running the Light Client.

**Reply by SpiralBlu** [reply]
Also, what partition is used for the storage on Linux?  So I know where to split my paritions.

**Reply by jgresham** [reply]
Hey @SpiralBlu, I'm glad to hear you tried it out already!

There are no rewards. Running a full node is viewed by many as providing a public good. That being said, I totally understand not being able to justify buying hardware specifically to run a full node. It is not cheap for many people. I wish there were some mechanisms to reward people who run full nodes. 

The node data is stored in a folder `NiceNode` in the default application data location for you operating system (ex. for me on Ubuntu this is `/home/<user>/.config/NiceNode`). Command `df` tell you which path a disk is used for.
